react-infinite/build.boot

```diff
@@ -21,17 +21,17 @@
 
 (deftask package []
   (comp
-   (download :url (str "https://github.com/seatgeek/react-infinite/archive/v" +lib-version+ ".zip")
-             :checksum "3c9e2b831e996b59a19b560be5e9f77a"
+   (download :url (str "https://github.com/seatgeek/react-infinite/archive/" +lib-version+ ".zip")
              :unzip true)
-
-   (sift :move {#"^react-infinite-(.*)/dist/react-infinite.js$" "cljsjs/react-infinite/development/react-infinite.inc.js"
-                #"^react-infinite-(.*)/dist/react-infinite.min.js$" "cljsjs/react-infinite/production/react-infinite.min.inc.js"})
-
+   (sift :move {#"^react-infinite-(.*)/dist/react-infinite.js$"
+                "cljsjs/react-infinite/development/react-infinite.inc.js"
+                #"^react-infinite-(.*)/dist/react-infinite.min.js$"
+                "cljsjs/react-infinite/production/react-infinite.min.inc.js"})
    (sift :include #{#"^cljsjs"})
-
-   (deps-cljs :name "react-infinite"
-              :requires ["react" "react-dom" "prop-types"])
+   (deps-cljs :name "cljsjs.react-infinite"
+              :requires ["cljsjs.react"
```

Comment:
The names without `cljsjs.` prefix are in fact correct. These names are used to match npm packages, so the libraries can use either cljsjs foreign-lib or npm modules, without changing their code (if the foreign-lib also supports global-exports). For backwards compatibility e.g. React provides both `react` and `cljsjs.react`. Packages using global-exports should depend the "new" name (`react`), and others can choose either.

As this package used to provide `react-infinite` I think it is best to provide the same name in new version, but we can also provide the "old" style name, this should do:

```
(deps-cljs :foreign-libs [{:file #"react-infinite.inc.js"
                           :file-min #"react-infinite.min.inc.js"
                           :provides ["react-infinite" "cljsjs.react-infinite"
                           :requires ["react" "react-dom" "prop-types"]
    :externs [#"react-infinite.ext.js"])
```

Reply:
https://github.com/cljsjs/packages/wiki/Unified-module-names-and-global-exports